daml-assistant/daml-helper/test/DA/Daml/Helper/Test/Deployment.hs

```diff
@@ -0,0 +1,128 @@
+-- Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+module DA.Daml.Helper.Test.Deployment (main) where
+
+import Data.List.Extra (isInfixOf,splitOn,intercalate)
+import System.Directory.Extra (withCurrentDirectory)
+import System.Environment.Blank (setEnv)
+import System.FilePath (searchPathSeparator,(</>))
+import System.IO.Extra (withTempDir,writeFileUTF8)
+import Test.Tasty (TestTree,defaultMain,testGroup)
+import Test.Tasty.HUnit (testCaseSteps)
+import qualified Data.Aeson as Aeson
+import qualified Data.Map as Map
+import qualified Data.Text as T
+import qualified Web.JWT as JWT
+
+import DA.Bazel.Runfiles (mainWorkspace,locateRunfiles)
+import DA.Test.Calling
+import DA.Test.Sandbox (mbSharedSecret,withSandbox,defaultSandboxConf)
+import DA.Test.Util (withEnv)
+import SdkVersion
+
+main :: IO ()
+main = do
+  -- We manipulate global state via the working directory
+  -- so running tests in parallel will cause trouble.
+  setEnv "TASTY_NUM_THREADS" "1" True
+  helper <- locateRunfiles (mainWorkspace </> "daml-assistant" </> "daml-helper")
+  damlc <- locateRunfiles (mainWorkspace </> "compiler" </> "damlc")
+  let path = intercalate [searchPathSeparator] ["/bin", "/usr/bin", damlc, helper]
```

Comment:
Why do we need `/bin`?

Reply:
It's gone. I change from using callProcess to callCommand instead.